src/cmd-build

```diff
@@ -216,31 +220,43 @@ fi
 
 /usr/lib/coreos-assembler/gf-anaconda-cleanup "$(pwd)"/"${img_base}"
 /usr/lib/coreos-assembler/gf-oemid "$(pwd)"/"${img_base}" "$(pwd)"/"${img_qemu}" qemu
-set +x
```

Comment:
no longer making things verbose?

Reply:
😄 the opposite. I'm leaving it verbose all the way through.

Reply:
oops - read it the wrong way